src/util.js

```diff
@@ -0,0 +1,42 @@
+import fs from "fs";
+import ERROR_MESSAGE from "./constant/errorMessage.js";
+
+export const isNumber = (num) => !Number.isNaN(Number(num));
+
+export const splitString = (input, separator) => input.split(separator);
+
+export const isInputWrappedWith = (input, wrapper) =>
+  input[0] === wrapper[0] && input[input.length - 1] === wrapper[1];
+
+export const getPromotionCount = (productCount, promoBundleSize, restCount = null) => {
+  if (restCount) return Math.floor((productCount + restCount) / promoBundleSize);
+  return Math.floor(productCount / promoBundleSize);
+};
+
+export const isPromotionShortage = ({ buy, get, promoBundleSize, productCount, restCount }) =>
+  productCount % promoBundleSize === buy && restCount > get;
+
+export const getDataFromFile = async (filePath) =>
+  fs.readFileSync(filePath, "utf8", (err, data) => {
+    if (err) {
+      throw new Error(ERROR_MESSAGE.fileReadError);
+    }
+    return data;
+  });
```

Comment:
제 생각에는 이 부분에 약간 오류가 있는 것 같습니다..! `fs.readFileSync()` 는 동기 함수여서 콜백을 사용할 필요가 없습니다.
혹시 `fs.readFile()`과 헷갈리신 건 아닌지 여쭙니다..! 참고 자료는 아래 첨부해두었습니다!
[fs.readFileSync()](https://nodejs.org/api/fs.html#fsreadfilesyncpath-options), [fs.readFile()](https://nodejs.org/api/fs.html#fsreadfilepath-options-callback)

만약 비동기로 처리하고 싶으시다면 `callback` 기반의 [fs.readFile()](https://nodejs.org/api/fs.html#fsreadfilepath-options-callback)나 `Promise` 기반의 [fsPromises.readFile()](https://nodejs.org/api/fs.html#fspromisesreadfilepath-options)을 사용하면 될 것 같습니다.
아니면, `fs.readFileSync()`를 사용해서 에러를 반환받고 싶다면 `try-catch` 로 `throw`를 잡는 게 좋을 것 같아요! 
[how-to-capture-no-file-for-fs-readfilesync](https://stackoverflow.com/questions/14391690/how-to-capture-no-file-for-fs-readfilesync)
정확하진 않지만 아래와 같은 방식으로 수정하면 될 것 같다는 생각이 들어요 :)
```suggestion
export const getDataFromFile = (filePath) => {
  try {
    return fs.readFileSync(filePath, "utf8");
  } catch (err) {
    throw new Error(ERROR_MESSAGE.fileReadError);
  }
};
```